extensions/cache/runtime/src/main/java/io/quarkus/cache/augmented/AugmentedCacheInvalidate.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.cache.augmented;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.enterprise.util.Nonbinding;
+import javax.interceptor.InterceptorBinding;
+
+/**
+ * @author Gwenneg Lepage
+ */
+@InterceptorBinding
+@Retention(value = RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE, ElementType.METHOD })
+public @interface AugmentedCacheInvalidate {
```

Comment:
The internal classes/interfaces/annotations should be in a `runtime` subpackage.